### PR TITLE
[mealie] Fix loading environment variables for backend

### PR DESCRIPTION
**Pull request by Jonnobrow**
**Description of the change**

Fix loading of environment variables for API component.

**Benefits**

Any env variables specified by user will be loaded.

**Possible drawbacks**

None.

**Applicable issues**

- N/A

**Additional information**

**Checklist** 
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [x] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

**Comment by Jonnobrow** [comment]
Seems like there was a temporary failure fetching the bitnami chart. Can a maintainer kick off another CI run please?

**Comment by bjw-s** [comment]
Retriggered the CI

**Review by Truxnell — APPROVED** [review]

File: charts/stable/mealie/templates/common.yaml [hunk 1]
```diff
@@ -14,6 +14,10 @@ additionalContainers:
     env:
       - name: API_PORT
         value: "9000"
+      {{- range $name, $value := .Values.api.env }}
+      - name: {{ $name }}
+        value: {{ $value }}
+      {{- end }}
```
Comment (Diaoul) [inline charts/stable/mealie/templates/common.yaml:20]: Does not work when passing down bool values as string as the quotes are removed.
See [here](https://discord.com/channels/673534664354430999/740734177279868949/982910175788826724)
e.g. with:
```yaml
ALLOW_SIGNUP: "false"
```
This throws:
> json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string

Comment (Jonnobrow) [reply charts/stable/mealie/templates/common.yaml:20]: Ah, should probably be `$value | quote` then. I can make that change later today potentially, but for now you can add in extra quotes to your values. E.g. `myValue: '"false"'`

Comment (Jonnobrow) [reply charts/stable/mealie/templates/common.yaml:20]: Resolve by #1606

**Pull request merged**